packages/mui-codemod/src/deprecations/divider-props/test-cases/expected.js

```diff
@@ -7,12 +7,8 @@ import { Divider as MyDivider } from '@mui/material';
 <MyDivider className="test" sx={{
   opacity: "0.6"
 }} />;
-<Divider className="test" sx={{
```

Comment:
Is this change expected? In case of `light` prop defined as variable,  current codemod adds sx irrespective of `light` value. Codemod in this pr  removes light prop and doesn't add sx.

I Don't have strong opinion on what should happen in case of light prop defined as variable, as we couldn't know value of it while running codemod

Reply:
These two cases (3rd and 4th) that changed correspond to:

```js
<Divider light={false} className="test" />;
<MyDivider light={false} className="test" />;
```

and not to (5th and 6th):

```js
<Divider light={light} className="test" />;
<MyDivider light={light} className="test" />;
```

Reply:
Yeah, I noticed that later. Hence removed comment. Sorry for confusion